### PR TITLE
New Fonts Added

### PR DESCRIPTION
**Pull request by Sammed-Littra**
Screenshot (220)
Screenshot (221)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -68,7 +76,7 @@
       body {
         margin: 0;
         padding: 0;
-        font-family: regular;
+        font-family: "Raleway", sans-serif;
```
Comment (vishakhg98) [inline public/index.html:79]: If we're going to only use Raleway font across entire or majority of website 
Add *{
font-family: "Raleway", sans-serif;
}

Then no need to add it individually for each classNames

File: src/WebSite Ui Components/DairyCard/DairyCard.css [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
 :root {
     --black: black;
-    --font:ProximaNova-SemiBold;
+    --font:'Raleway', sans-serif;;
```
Comment (vishakhg98) [inline src/WebSite Ui Components/DairyCard/DairyCard.css:4]: 2 semi colons

File: public/index.html [hunk 3]
```diff
@@ -153,25 +161,25 @@
...
+          url("https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;700&display=swap")
             format("woff");
       }
 
       @font-face {
-        font-family: "ProximaNova-Bold";
+        font-family: "Raleway", sans-serif;
         font-display: swap;
         src: url("https://cdn.theluxurycloset.com/themes/mobile_new/css/fonts/hinted-ProximaNova-Bold.woff2")
             format("woff2"),
```
Comment (vishakhg98) [inline public/index.html:185]: Does above @font-face lines usefull? as youre importing font using <link> tag.

**Comment by Sammed-Littra** [comment]
Ok vishak I will make changes

**Review by aaku14n — APPROVED** [review]

**Pull request merged**